docs/admin/troubleshooting_updates.rst

```diff
@@ -107,16 +107,15 @@ Note that ``dom0`` and ``apply_dom0`` are separate steps.
    security-sensitive, and may require a reboot to take
    effect.
    
-GPG check failures
-^^^^^^^^^^^^^^^^^^
-
-If at any point you see a message that the GPG check failed,
-and you see an indication that a key has expired, your system
-may be trying to use an old copy of the SecureDrop Release Signing
-Key. 
-
-To remove the expired key and replace it with the new, valid key, you
-can perform the following steps:
+Expired SecureDrop Signing Key (2021)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
I hate to do this to you, but I don't think (2021) should be in the title. The new master key fingerprint was generated in 2021, but the specific pubkey was valid 2022-july to 2023- july. I think "Expired SecureDrop Signing Key" is fine.

In the next paragraph I'd start with

"If after running `sudo qubes-dom0-update` as described above, the update fails and the terminal console displays the following message:
```
1. Certificiate [sic] 188EDD3B7B22E6A3 invalid: certificate is not alive
    because: The primary key is not live
    because: Expired on 2023-07-04T10:52:20Z
2. Key 188EDD3B7B22E6A3 invalid: key is not alive
    because: The primary key is not live
    because: Expired on 2023-07-04T10:52:20Z
[...]
Error: GPG check FAILED
```
the system is trying to use an old copy.... (etc)

Reply:
and yes there is a typo in the console output ("certificiate") and we should leave the typo in the docs